src/System.Private.CoreLib/shared/System/IO/StreamWriter.cs

```diff
@@ -1028,16 +1018,6 @@ public override Task FlushAsync()
             return task;
         }
 
-        private int CharPos_Prop
```

Comment:
I guess these properties were workaround for slow field access with remoting in .NET Framework.